logic/src/androidMain/kotlin/com/wire/kalium/logic/feature/call/CallManagerImpl.kt

```diff
@@ -235,13 +226,6 @@ class CallManagerImpl internal constructor(
 
     override suspend fun rejectCall(conversationId: ConversationId) = withCalling {
         callingLogger.d("[$TAG][rejectCall] -> ConversationId: [$conversationId]")
-        val conversationType = callRepository.getLastCallConversationTypeByConversationId(conversationId = conversationId)
-
-        callingLogger.d("[$TAG][rejectCall] -> ConversationType: [$conversationType]")
-        callRepository.updateCallStatusById(
-            conversationIdString = conversationId.toString(),
-            status = if (conversationType == Conversation.Type.GROUP) CallStatus.STILL_ONGOING else CallStatus.CLOSED
-        )
```

Comment:
By removing the update of call status here, the user will then be waiting for this flow:

`user is in a call -> user ends the call -> [request to UseCase -> Repository -> Network/AVS] -> [response from AVS -> Repository -> UseCase] -> User is alerted and only then the screen is closed`

By having the call to update the call status here directly, we don't need to wait on Network/AVS response and only then closing the screen, so the user doesn't need to wait the whole process for the call screen to close.

By removing it did you see any drastic change on the wait time for the call screen to be closed/dismissed?